great_expectations/util.py

```diff
@@ -79,14 +74,14 @@
     # needed until numpy min version 1.20
     import numpy.typing as npt
 
-    from great_expectations.alias_types import PathStr
-    from great_expectations.data_context import CloudDataContext, FileDataContext
     from great_expectations.data_context.data_context.abstract_data_context import (
         AbstractDataContext,
     )
-    from great_expectations.data_context.types.base import DataContextConfig
 
 
+# Make get_context importable from great_expectations.util but encapsulate it in its own file
+get_context = _get_context
+
```

Comment:
Do we need to import this here at all?
As long as it's imported in `great_expectations.__init__.py` it should be available.